### PR TITLE
Etf md only source config

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**
Documentation describing the metadata only workflow is added.
Documentation describing the ETF validator is added, and multiple related pages are updated to remove outdated content related to the GDI-Test suite.
Documentation describing the use of WFS and databases is added.

Once the style of hc-test is reverted, or the functionality enters production, I will update the screenshots.

### CODE REVIEW

**Comment by Kate-Lyndegaard** [comment]
@JohannaOtt @stempler Once the English version of this PR is agreed upon, I will create the German version based on that.

**Comment by Claudia-WE** [comment]
@Kate-Lyndegaard  Hi Kate,
@JohannaOtt has asked me to take over the review of this pull request. If you agree, please change the assigned reviewer.

**Comment by Kate-Lyndegaard** [comment]
@Claudia-WE Super, I added you as a reviewer.

File: docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md
Comment (Claudia-WE) [inline docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md:1]: > hale»connect currently supports the creation of dataset metadata, without requiring the upload of associated data. The metadata only workflow is not available for service metadata.

hale»connect currently supports the creation of dataset metadata, without requiring the upload of associated data. 
This **metadata only workflow** is not available for service metadata.

File: docs/references/themes/2018-04-03-reference-themes-validation.md
Comment (Claudia-WE) [inline docs/references/themes/2018-04-03-reference-themes-validation.md:1]: LGTM

File: docs/setup-hc/config-source-data/2015-02-09-WFS.md
Comment (Claudia-WE) [inline docs/setup-hc/config-source-data/2015-02-09-WFS.md:1]: 3.	Select base theme. In _this_ first step, decide whether you want to re-use an existing theme configuration by deriving your new theme from the selected one _or if you decide to not inherit one_. You will be able to edit the configuration later on. Click «Continue»
4.	Select name and type. In _this_ second step,
5.	Add feature types. In _this_ third step, click «Add new type» [...]When you are finished, click «Continue». _If you have inherited a start configuration in the processing step 1 (Select base theme) the inherited features are already allocated to the theme._

File: docs/references/themes/2018-04-03-reference-themes-validation.md
Comment (Claudia-WE) [inline docs/references/themes/2018-04-03-reference-themes-validation.md:1]: LGTM

File: docs/setup-hc/config-source-data/2015-02-09-WFS.md
Comment (Claudia-WE) [inline docs/setup-hc/config-source-data/2015-02-09-WFS.md:1]: 3.	Select base theme. In _this_ first step, decide whether you want to re-use an existing theme configuration by deriving your new theme from the selected one _or if you decide to not inherit one_. You will be able to edit the configuration later on. Click «Continue»
4.	Select name and type. In _this_ second step,[...]
5.	Add feature types. In _this_ third step, click «Add new type» to add feature types to your theme. Click on the » button to select a schema that contains the feature types you would like to add. Next, click the » button to select a single feature type or click «Add all» to add all of the listed feature types to your theme. When you are finished, click «Continue». _If you have inherited a start configuration in the processing step 1 (Select base theme) the inherited features are already allocated to the theme._

File: docs/setup-hc/config-source-data/2015-02-10-database.md
Comment (Claudia-WE) [inline docs/setup-hc/config-source-data/2015-02-10-database.md:1]: @Kate-Lyndegaard I am irritated when I compared the MD file name with the content. Is that correct ?

**Review by Claudia-WE — COMMENTED** [review]
Hi @Kate-Lyndegaard 
I finished my review. I hope my suggestions are supportive.

File: docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md
Comment (Kate-Lyndegaard) [reply docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md:1]: @Claudia-WE I know that you encourage use of bold and other format choices to emphasize meaning, my only comment is that this is not done anywhere else in the documentation currently, so I hesitate to begin randomly now.

Regarding adding feature types- no, you do not need to add feature types in the workflow, however it is stated here that the user should just accept the default and click continue.

File: docs/references/data/2018-03-11-reference-data-overview.md
[the remark this answers is not in the document]
Comment (Kate-Lyndegaard) [reply docs/references/data/2018-03-11-reference-data-overview.md:1]: @Claudia-WE I am not sure what system you are working on.. You can see it on the test system.

File: docs/setup-hc/config-source-data/2015-02-09-WFS.md
Comment (Kate-Lyndegaard) [reply docs/setup-hc/config-source-data/2015-02-09-WFS.md:1]: @Claudia-WE Unfortunately, I don't know what your comments are referring to. I cannot find them in [docs/setup-hc/config-source-data/2015-02-09-WFS.md](https://github.com/wetransform/hc-documentation/pull/55/files/0d09a13b93270a5b7e816e5a87a216574ba8a262#diff-3eaae012c2c2f83efb8c8b7f854797a136cd5f9cbc17fee8640db1ed934dd045)

File: docs/setup-hc/config-source-data/2015-02-10-database.md
Comment (Kate-Lyndegaard) [reply docs/setup-hc/config-source-data/2015-02-10-database.md:1]: @Claudia-WE Unfortunately, I am not sure what you mean. Maybe we can have a call to clarify.

File: docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- "help-page-subcategory-datasetworkflowcreatemetadata"
+layout: help-detail
+language: en
+---
+
+
+hale»connect currently supports the creation of dataset metadata, without requiring the upload of associated data. The metadata only workflow is not available for service metadata.
+
+## Create metadata only theme ##
+
```
Comment (Claudia-WE) [inline docs/create-manage-datasets/create-metadata/2015-01-10-metadata-create.md:14]: test

**Review by Claudia-WE — APPROVED** [review]

**Pull request merged**